zou/app/blueprints/persons/resources.py

```diff
@@ -75,16 +77,9 @@ def post(self):
                 "limit": config.USER_LIMIT,
             }, 400
         else:
-            person = persons_service.create_person(
```

Comment:
You should keep this., it ensures that data is properly used.

Reply:
It's fixed